Header header
uint16 proto
bool is_ip
bool is_udp
bool ieee80211_to_ds
bool ieee80211_is_retry
bool ieee80211_is_beacon
int8 retries
float32 rate
uint16 channel
int8 rssi
uint16 snap_type
uint16 udp_port
uint16 ieee80211_duration
uint16 ieee80211_seq
uint8 antenna
int64 mac_time
uint16 len
int64 serial
int8 sender
string addr1
string addr2
string addr3
uint8[] data

